src/js/main.js

```diff
@@ -0,0 +1,18 @@
+$.getJSON('/../../api/brewery-db/zombie-killer.json')
+ .then(function(zombieKiller){
+   console.log(zombieKiller);
+
+   $beerTitle = $('.beername');
+   $beerTitle.text(zombieKiller.data.name);
+
+ });
+
+$.getJSON('/../../api/youtube/b-nektar.json')
+.then(function(bnektar){
+  console.log(bnektar);
+
+  $vidTitle = $('#nektar');
+  $vidTitle.text(bnektar.title);
+
+
```

Comment:
@al-the-x

I am trying to get the video to show, and so far only get a white box. 
I have downloaded the `JSON` file, and was able to replace the title. I am also able to embed the video directly into the site from the browser using the embed button on youtube, but do not feel that is the right way. other options I have tried are `$vidPlayer.attr(bnektar.html)` ,  `$vidPlayer.attr(bnektar.html, bnektar.thumbnail_url)`,  `$vidPlayer.attr(bnektar.thumbnail_url)`. so far none has worked. I have also checked ; [HTML GOODIES](http://www.htmlgoodies.com/html5/client/how-to-embed-video-using-html5.html#fbid=CMPkvb99egB), [stackoverflow](http://stackoverflow.com/questions/10066638/get-youtube-information-via-json-for-single-video-not-feed-in-javascript) to get the JSON file. youtube developers, etc most of which offer using a javascript model, would that be the right way to go?
Thanks, 
Nick